.gitignore

```diff
@@ -10,7 +10,6 @@ client/build
 server/*.pdf
 server/pdfs/*
 server/db/.migrate
-server/scripts/xlsx
```

Comment:
I'd like to keep this gitignore - this exists to prevent accidental commit of prod files, which were historically kept local to the scripts folder for upload

Preference would be to create a new folder server/tests/files and put the xlsx files for testing there so it's super clear what they are, and they aren't in with prod code vs test code